app/assets/stylesheets/home.scss

```diff
@@ -89,3 +89,18 @@
   max-height: 700px;
   max-width: 100%;
 }
+
+@media (max-width:765px) {
+  .row1 {
```

Comment:
Please chose better name instead `.row1`

Reply:
I changed to button-homepage. 
Please check it @tachyons